File: TrabajoSep04/Cpp/Circulo.cpp
#include <iostream>
#include <math.h>


class Circulo {
public:
    double radio;
    double area;
    double perimetro;
    double pi = M_PI;

    Circulo (double newRadio) {
        radio = newRadio;
        area = calcularArea();
        perimetro = calcularPerimetro();
    };

    double calcularArea(){
        return pi * radio * radio;
    }

    double calcularPerimetro() {
        return pi * 2 * radio;
    }
    void nuevoRadio(double newRadio){
        radio = newRadio;
    }
};

File: TrabajoSep04/Java/Circulo.java
public class Circulo {
    double radio;
    double area;
    double perimetro;
    double PI = Math.PI;

    Circulo(double radio){
        this.radio = radio;
        this.area = this.calcularArea();
        this.perimetro = this.calcularPerimetro();
    }

    double calcularArea() {
        return PI * this.radio * this.radio;
    }
    double calcularPerimetro() {
        return PI * 2 * this.radio;
    }

    double cambiarRadio(double radio){
        return this.radio = radio;
    }

}

